src/workflows/airqo_etl_utils/bigquery_api.py

```
@@ -1383,26 +1382,26 @@ def fetch_raw_site_data_for_forecast_jobs(
-            consolidated_table=f"`{self.consolidated_data_table}`",
+            hourly_measurements_table=f"`{self.hourly_measurements_table}`",
             sites_table=f"`{self.sites_table}`",
             start_date=str(start_date),
             end_date=str(end_date),
-            min_hours=2,
+            min_hours=min_hours,
         )
 
         try:
             return self.execute_data_query(query=query)
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)